test_ir.py

```diff
@@ -37,5 +37,68 @@ class DisplaySystem(jkt.KernelTests):
     {'code': '1:3', 'mime': {'text/plain':'[1] 1 2 3'}},
     ]
 
+class AbstractIRKernel(jkt.KernelTests):
```

Comment:
My intention was that the kernel was restarted for each test... as I play with options it seems to do that.